src/RevMemCtrl.cc

```diff
@@ -1294,6 +1291,8 @@ void RevBasicMemCtrl::handleReadResp(StandardMem::ReadResp* ev){
         if( isAMO ){
           handleAMO(op);
         }
+        bool *Hazard = op->getHazard();
+        *Hazard = false;
         delete op;
```

Comment:
So we allocated this in RevProc, and we will also be calling `destroyHazard` from there - do we want to delete this value here or just set to nullptr and have it cleaned up in RevProc?

Reply:
Agreed - revProc allocates so RevProc has to destroy